src/applications/facility-locator/utils/helpers.js

```diff
@@ -80,6 +80,42 @@ export const areGeocodeEqual = (pos1, pos2) => {
   return (pos1.latitude === pos2.latitude) && (pos1.longitude === pos2.longitude);
 };
 
+/**
+ * Compares two geographic bounding boxes to determine if they are equal.
+ *
+ * A bounding box is expected to be of the shape
+ *   [lat1, long1, lat2, long2]
+ *
+ * @param {number[]} box1 The first bounding box's coords
+ * @param {number[]} box2 The second bounding box's coords
+ */
+export const areBoundsEqual = (box1, box2) => {
+  if (!box1 || !box2 || box1.length !== 4 || box2.length !== 4) {
```

Comment:
What's the `4` mean here?

Reply:
It looks like the length 4 is checking that box1 and box2 have the expected 4 coordinates in the bounding box; if not 4 the coordinates are invalid and the method returns false

Reply:
Yep, exactly as Ben said

Reply:
That might be good, but I think it might be more helpful later if there were a comment saying why it needed to be 4 exactly. Something as simple as `// Bounding boxes need 4 coordinates to be valid` should do the trick.

Reply:
A comment about how bounding boxes need 4 coordinates to be considered valid probably will suffice; IMO I don't see a need to use a variable because we aren't doing anything else with that number

Reply:
Sounds good, I'll be sure to add the comment in 👍